content/download.html.haml

```diff
@@ -9,45 +9,92 @@
 
-                %p Alternatively, if you have a servlet container that supports Servlet 2.4/JSP 2.0, such as Glassfish v2, Tomcat 5 (or any later versions), then you can run them as services, and deploy jenkins.war as you would any other war file. Container specific documentation is available if you choose this route.
-            .sub-block
-              %h4 Windows Installation
-              %p If you're running on Windows it is good to run Jenkins as a service so it starts up automatically without requiring a user to log in. The easiest way is to run the Windows installer, linked from Jenkins' homepage. This also has the advantage of being easier to automate.
+          %p Since Jenkins was written to work on unix-like platforms, some parts assume the presence of unix-utilities. It is advised to install these as well on Windows. Install UnxUtils (this includes a shell that seems to work with forward and backwards slashes and does globbing correctly)(UnxUtils does not download), put it in the Windows PATH , and copy sh.exe to C:\bin\sh.exe (or whichever drive you use) to make shebang lines work. This should get you going. If UnxUtils gives you trouble (Fork Failed Errors), try Win-Bash.
 
-              %p The manual way is to follow Installing Jenkins as a Windows service. Alternatively, you can install a servlet container like GlassFish and Tomcat, which can run as a service by itself, and then deploy Jenkins to it.
+          %ul
+            %li
+              %a{ href: "https://wiki.jenkins-ci.org/display/JENKINS/Installing+Jenkins+as+a+Windows+service", title: "Install Service"} Installing Jenkins as a Windows service
+          %p
+          %br
 
-              %p Since Jenkins was written to work on unix-like platforms, some parts assume the presence of unix-utilities. It is advised to install these as well on Windows. Install UnxUtils (this includes a shell that seems to work with forward and backwards slashes and does globbing correctly)(UnxUtils does not download), put it in the Windows PATH , and copy sh.exe to C:\bin\sh.exe (or whichever drive you use) to make shebang lines work. This should get you going. If UnxUtils gives you trouble (Fork Failed Errors), try Win-Bash.
+          %h3 Setup Wizard
+          %p Jenkins 2.0 is no longer installed completely open and accessible to the world. Instead Jenkins is initially configured to be more secure. Jenkins can no longer be accessed without a username and password and open ports are limited. During the initial run of Jenkins a security token is generated and printed in the console log:
+          %pre.code
+            *************************************************************
+            *************************************************************
+            *************************************************************
+            %p
```

Comment:
This is unnecessary inside of a `%pre` and I _believe_ this will actually be rendered instead of inserting a `<p/>` tag